content/documentation/pages/5-feature-guides/2-batch/4-monitoring.md

```diff
@@ -0,0 +1,170 @@
+```
+
+- Starting Docker Compose
+
+In the directory where you downloaded `docker-compose-influxdb.yml`, start the system, by running the following commands:
+
+```bash
+export DATAFLOW_VERSION=%dataflow-version%
+export SKIPPER_VERSION=%skipper-version%
+docker-compose -f ./docker-compose-influxdb.yml up
+```
+
+Now that docker compose is up, you can access the [Spring Cloud Data Flow Dashboard](http://localhost:9393/dashboard). Also you can reach the Grafana dashboard at http://localhost:3000 using the user: admin, password: admin credentials.
+
+Now you can deploy a custom Task application (`task-demo-metrics`) and define two tasks (`task1` and `task2`):
+
+```bash
+dataflow:>app register --name myTask --type task --uri https://github.com/tzolov/task-demo-metrics/raw/master/apps/task-demo-metrics-0.0.1-SNAPSHOT.jar
+
+dataflow:>task create --name task1 --definition "myTask"
+dataflow:>task create --name task2 --definition "myTask"
+```
+
+Launch the tasks several times:
+
+```bash
+dataflow:>task launch --name task1
+dataflow:>task launch --name task2
+```
+
+In the [DataFlow task execution UI](http://localhost:9393/dashboard/#/tasks/executions) you should see list like this:
+![SCDF Task Execution](images/SCDF-metrics-task-execution-view.png)
+
+You should see dashboards similar to those shown in the following image:
+
+![SCDF Task Grafana InfluxDB](images/SCDF-metrics-grafana-task.png)
+
+## Cloud Foundry
+
+This section describes how to set up InfluxDB for Cloud Foundry.
+
+### InfluxDB
+
+You can follow the general [Manifest based installation on Cloud Foundry](http://localhost:8000/docs/installation/cloudfoundry/cf-cli/#manifest-based-installation-on-cloud-foundry) instructions for installing `Skipper` and `DataFlow` on Cloud Foundry.
+To enabling the Task metrics integration you need to extend the [DataFlow manifest](http://localhost:8000/docs/installation/cloudfoundry/cf-cli/#installing-using-a-manifest) with following `SPRING_APPLICATION_JSON` variable:
+
+```json
+{
+  "spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.enabled": true,
+  "spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.db": "yourinfluxdb",
+  "spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.autoCreateDb": false,
+  "spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.uri": "https://your-influx-uri:port",
+  "spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.userName": "influxusername",
+  "spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.password": "******",
+  "spring.cloud.dataflow.grafana-info.url": "https://yor-grafana-uri:443"
+}
+```
+
+Check the [Influx Actuator properties](https://docs.spring.io/spring-boot/docs/2.2.0.M4/reference/html/#actuator-properties) for further details about the `management.metrics.export.influx.XXX` properties.
```

Comment:
Did you mean to point to `2.2.0.M4` docs?

Reply:
Given that the Spring Batch monitoring is available only from Spring Boot 2.2.0.M4 onwards i thought that i should point to this documentation version, at least until 2.2.0 GA is available

Reply:
Sounds good.